modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -551,31 +551,19 @@ class SegmentEdit extends Component {
 										<div className="btn-group">
 											<div className="btn-group-item mr-2">
 												<ClayToggle
```

Comment:
I think the aria-label is not necessary, I've removed it and the label 'Edit Segment' still pop-up when I hover over or focus on the toggle. Maybe with the title is enough

Reply:
Hi! The PEDS Team said that adding aria-label and title always ensures more consistent reading so I think it is better to leave them both.